src/main/java/pl/targosz/homework/FileReverse.java

```diff
@@ -0,0 +1,18 @@
+package pl.targosz.homework;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+
+public class FileReverse {
+    public static void main(String[] args) throws IOException {
+
+        File file = new File ("src/main/java/pl/targosz/homework/resources/data.txt");
```

Comment:
Nigdzie tego nie używasz, po co Ci to?

Reply:
I to do usunięcia:
![image](https://user-images.githubusercontent.com/17989854/136224976-30c5f68e-302c-4be1-a6ca-491486750088.png)